src/backend/main.ts

```diff
@@ -166,6 +172,62 @@ export async function bootstrapWorkspace(opts: {
     yDoc.clientID = clientID
   }
 
+  const appPrefixName = appPrefix.toString()
+  const snapshotName = appPrefix.append('32=snapshot').toString()
+  const localYJSUpdate = await persistStore.get('localSnapshot')
+
+  const localTimestamp = await persistStore.get('snapshotTimestamp')
+
+  const timestampInterval = 86400000 //24hr
+  if (!localYJSUpdate || !localTimestamp || Date.now() - NNI.decode(localTimestamp) > timestampInterval) {
+    const interest = new Interest(snapshotName, Interest.CanBePrefix, Interest.MustBeFresh)
+    try {
+      const data = await consume(interest)
+      const targetName = data.name.getPrefix(-1)
+      // /grpPrefix/32=snapshot/54=<vector>/
+      let snapshotData = await fetch(targetName, {
+        verifier: certStorage.verifier, // we have access to the verifier
+        modifyInterest: { mustBeFresh: true },
```

Comment:
I remember we said to remove `mustBeFresh` if it is not a prefix match, because it blocks repo from serving the content.

Reply:
I left mustBeFresh on for testing. If tests go well we should remove it.